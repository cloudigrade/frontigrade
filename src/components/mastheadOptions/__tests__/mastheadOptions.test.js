import React from 'react';
import { shallow } from 'enzyme';
import MastheadOptions from '../mastheadOptions';

describe('MastheadOptions Component', () => {
  it('should render', () => {
    const props = {
      user: { username: 'Admin' },
      logoutUser: () => {},
      showAboutModal: () => {}
    };

    const component = shallow(<MastheadOptions {...props} />);

    expect(component).toMatchSnapshot();
  });
});
