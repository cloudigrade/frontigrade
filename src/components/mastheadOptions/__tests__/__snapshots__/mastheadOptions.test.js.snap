// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MastheadOptions Component should render 1`] = `
<nav
  className="collapse navbar-collapse"
>
  <ul
    className="navbar-iconic nav navbar-nav navbar-right"
  >
    <Uncontrolled(Dropdown)
      componentClass="li"
      id="help"
    >
      <DropdownToggle
        bsClass="dropdown-toggle"
        bsRole="toggle"
        className="nav-item-iconic"
        open={false}
        useAnchor={true}
      >
        <Icon
          name="help"
          type="pf"
        />
      </DropdownToggle>
      <DropdownMenu
        bsClass="dropdown-menu"
        bsRole="menu"
        pullRight={false}
      >
        <MenuItem
          bsClass="dropdown"
          disabled={false}
          divider={false}
          header={false}
          onClick={[Function]}
        >
          About
        </MenuItem>
      </DropdownMenu>
    </Uncontrolled(Dropdown)>
    <Uncontrolled(Dropdown)
      componentClass="li"
      id="user"
    >
      <DropdownToggle
        bsClass="dropdown-toggle"
        bsRole="toggle"
        className="nav-item-iconic"
        open={false}
        useAnchor={true}
      >
        <Icon
          name="user"
          type="pf"
        />
         
        Admin
      </DropdownToggle>
      <DropdownMenu
        bsClass="dropdown-menu"
        bsRole="menu"
        pullRight={false}
      >
        <MenuItem
          bsClass="dropdown"
          disabled={false}
          divider={false}
          header={false}
          onClick={[Function]}
        >
          Log out
        </MenuItem>
      </DropdownMenu>
    </Uncontrolled(Dropdown)>
  </ul>
</nav>
`;
